test/createExamples.js

```diff
@@ -0,0 +1,76 @@
+        developerName: null,                      // Your (or your developer's) name. `string`
+        developerURL: null,                       // Your (or your developer's) URL. `string`
+        dir: "auto",                              // Primary text direction for name, short_name, and description
+        lang: "en-US",                            // Primary language for name and short_name
+        background: "#fff",                       // Background colour for flattened icons. `string`
+        theme_color: "#fff",                      // Theme color user for example in Android's task switcher. `string`
+        appleStatusBarStyle: "black-translucent", // Style for Apple status bar: "black-translucent", "default", "black". `string`
+        display: "standalone",                    // Preferred display mode: "fullscreen", "standalone", "minimal-ui" or "browser". `string`
+        orientation: "any",                       // Default orientation: "any", "natural", "portrait" or "landscape". `string`
+        scope: "/",                               // set of URLs that the browser considers within your app
+        start_url: "/?homescreen=1",              // Start URL when launching the application from a device. `string`
+        version: "1.0",                           // Your application's version string. `string`
+        logging: false,                           // Print logs to console? `boolean`
+        pixel_art: false,                         // Keeps pixels "sharp" when scaling up, for pixel art.  Only supported in offline mode.
+        loadManifestWithCredentials: false,       // Browsers don't send cookies when fetching a manifest, enable this to fix that. `boolean`
+        icons: {
+            android: true,              // Create Android homescreen icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            appleIcon: true,            // Create Apple touch icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            appleStartup: true,         // Create Apple startup images. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            coast: true,                // Create Opera Coast icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            favicons: true,             // Create regular favicons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            firefox: true,              // Create Firefox OS icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            windows: true,              // Create Windows 8 tile icons. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+            yandex: true                // Create Yandex browser icon. `boolean` or `{ offset, background, mask, overlayGlow, overlayShadow }`
+        }
+    },
+    callback = function (error, response) {
```

Comment:
Not super important, but I'd expect an arrow function here instead of the more verbose `function` syntax.

Reply:
Updated the callback function to the arrow syntax. Left the error callback in the fswriteFile methods untouched for now. 
The file is just to manually create sample files on the fly and has no deeper meaning.